src/upload.ts

```diff
@@ -92,4 +95,13 @@ export class Uploader {
     async waitUploads (): Promise<void> {
         await Promise.all(this._uploads);
     }
+
+    async uploadScreenshot (basePath: string, postfix: string): Promise<string | undefined> {
```

Comment:
[minor] Since we use this method only for uploading layout testing screenshots I'd rather rename it to `uploadLayoutTestingArtifacts`